### PR TITLE
Add SQL support section to Pinot connector

### PR DESCRIPTION
**Pull request by Jessie212**

### CODE REVIEW

File: docs/src/main/sphinx/connector/pinot.rst [hunk 1]
```diff
@@ -114,3 +114,19 @@ Pinot                        Trino
...
+the Pinot database. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/drop-schema`
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/pinot.rst:132]: I think all statements aren't yet supported and they are ongoing in #7164, #7162, #7160
https://github.com/trinodb/trino/blob/master/plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotMetadata.java

Comment (jhlodin) [reply docs/src/main/sphinx/connector/pinot.rst:132]: @Jessie212 You can remove all these statements and use the read-only wording like in Druid

**Review by ebyhr — APPROVED** [review]

**Review by rosewms — APPROVED** [review]

File: docs/src/main/sphinx/connector/pinot.rst [hunk 2]
```diff
@@ -114,3 +114,11 @@ Pinot                        Trino
...
 ==========================   ============
 
+.. _pinot-sql-support:
+
+SQL support
+-----------
+
+The connector provides :ref:`globally available <sql-globally-available>` and
+:ref:`read operation <sql-read-operations>` statements to access data and
+metadata in the Pinot database.
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/pinot.rst:124]: Should be:

"...data and metadata in Pinot."

As opposed to "the Pinot Database"

**Review by jhlodin — APPROVED** [review]
LGTM with quick comment

File: docs/src/main/sphinx/connector/pinot.rst (hunk 2 above)
Comment (Jessie212) [reply docs/src/main/sphinx/connector/pinot.rst]: Applied.

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**